portfolio/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```

Comment:
all of the files in `portfolio/target/...` should be excluded from commits, since they're generated as part of the build. let's add target/ to gitignore.

Reply:
It looks like target/ is already in gitignore... I think I accidentally moved a file out of target and then had to move it back in which is why this happened